src/index.coffee

```diff
@@ -109,7 +109,7 @@ module.exports = (projectConfig={}) ->
       .pipe(source(config.browserify.entries[0]))
       .pipe(buffer())
       .pipe(ugly())
-      .pipe(concat('app.js'))
+      .pipe(concat('development-bundle.js'))
```

Comment:
Under which circumstances do we serve this? Only local development?

Reply:
Yeah, we only use gulp for local development.